tests/integration/integration/login/login-en.spec.js

```diff
@@ -0,0 +1,39 @@
+import { Selector } from "testcafe";
+import { googleAuthUser } from "../../helpers/loginByGoogle";
+import {
+  selectLocale,
+  selectLocaleFromDialog,
+} from "../../helpers/selectLocale";
+
+fixture("Login - English")
+  .page(`${process.env.BROWSERSTACK_BASE_URL}/login`)
+  .beforeEach(async (t) => {
+    await selectLocale("en");
+  });
+
+test("sees the login page", async (t) => {
```

Comment:
I would recommend segregating the parts about elements loading in the login page and the actual login step - also, when you do segregate the login step, do specifically mention that it is google log in.

Reply:
Done. The only thing here is when viewing in BrowserStack, I see the different `test()` as a separate run, having it's separate video recording. I believe it should be fine.